.github/BECOMING_A_CORE_CONTRIBUTOR.md

```diff
@@ -51,7 +51,7 @@ Lightning should reflect the broader community it serves. As such we should have
 
 ### Community
 
-We have an active [Slack](https://www.pytorchlightning.ai/community) community, where questions are asked daily.
+We have an active [Discord](https://discord.gg/VptPCZkGNa) and [Slack](https://www.pytorchlightning.ai/community) community, where questions are asked daily.
```

Comment:
```suggestion
We have an active [Discord](https://discord.gg/VptPCZkGNa) community, where questions are asked daily.
```

Reply:
Let's remove Slack. The people that are in Slack today won't check back here anyway